Firebase/Messaging/FIRMessagingReceiver.m

```diff
@@ -28,28 +27,15 @@
 
 static NSString *const kUpstreamMessageIDUserInfoKey = @"messageID";
 static NSString *const kUpstreamErrorUserInfoKey = @"error";
-/// "Should use Messaging delegate" key stored in NSUserDefaults
-NSString *const kFIRMessagingUserDefaultsKeyUseMessagingDelegate =
-    @"com.firebase.messaging.useMessagingDelegate";
-/// "Should use Messaging Delegate" key stored in Info.plist
-NSString *const kFIRMessagingPlistUseMessagingDelegate =
-    @"FirebaseMessagingUseMessagingDelegateForDirectChannel";
 
 static int downstreamMessageID = 0;
 
-@interface FIRMessagingReceiver ()
-@property(nonatomic, strong) GULUserDefaults *defaults;
-@end
-
 @implementation FIRMessagingReceiver
 
 #pragma mark - Initializer
 
-- (instancetype)initWithUserDefaults:(GULUserDefaults *)defaults {
+- (instancetype)init {
```

Comment:
Since this is an empty `init` call, we should just be able to remove it (although please confirm that the compiler doesn't complain - it shouldn't since `NSObject` implements it but just in case).